src/node_dotenv.h

```diff
@@ -13,6 +13,10 @@ namespace node {
 class Dotenv {
  public:
   enum ParseResult { Valid, FileError, InvalidContent };
+  struct env_file_data {
+    std::string path;
```

Comment:
No need to make an additional copy, let's use std::string_view and trust the lifecycle of the owned string:

```suggestion
    std::string_view path;
```

Reply:
I'll take care of it once pull request lands

Reply:
Thanks, much appreciated! I tried giving it a go, but being new to C++ I couldn't quite figure it out. I'm sure it's quite simple, too 🥲